src/drivers/bootloaders/CMakeLists.txt

```diff
@@ -47,6 +47,9 @@ if(NOT "${PX4_BOARD_LABEL}" MATCHES  "canbootloader")
 	set(HW_MINOR ${uavcanblid_hw_version_minor})
 	set(SW_MAJOR ${uavcanblid_sw_version_major})
 	set(SW_MINOR ${uavcanblid_sw_version_minor})
+	if("${uavcanbl_padding}" STREQUAL "")
+		set(uavcanbl_padding 4)
```

Comment:
Would simply padding by 8 everywhere work?

Reply:
Nope H7 is 32, NXP is 8 we will hit a 16. I have tried to get the linker to pad, but for some reason it is not fool proof.